_data/developer.yml

```diff
@@ -129,45 +131,44 @@ websites:
     img: cmtelecom.png
     tfa:
       - sms
-      - phone
-      - software
-    doc: https://help.onlinesmsgateway.com/en/accounts-and-payments/how-do-i-enable-two-factor-authentication
+      - proprietary
```

Comment:
I don't see any mentions of any app.

Reply:
I tried this by myself:
![cmtelecom-tfa](https://user-images.githubusercontent.com/520841/71326919-a57f4d80-2501-11ea-8a8a-846ccf047186.png)
The website did not accept my phone number (for whatever reason), so I could not check whether they use regular TOTP. However, the "CM Authenticator" app was not able to scan the TOTP QR codes that I already have, so I would classify this as "proprietary".